docs/adr/0012-high-level-design-infra-cluster.md

```diff
@@ -0,0 +1,119 @@
+*   Ilana’s cluster - a 4 node BM cluster that serves for the ‘CNV on MOC’ initiative and was installed using the IPI installer.
+*   All in one cluster - single node cluster running on a VM, with ACM as a managing component for Ilana's cluster.
+*   Currently all of our infrastructure environment is consistent with Bare metal Servers (the All in One cluster is running on a VM in the provisioner server.
+*   The Environment has a constant connection to the internet.
+
+The current system offers users a non-production environment to run Kubernetes workload, all the clusters are not prominent and are changed/modified weekly.
+
+
+## BASIC ASSUMPTIONS
+
+
+*   In this solution we are going to use only products that are provided by Red Hat or will be provided by Red Hat in the near future.
+*   We are using a BM only environment which resides in MOC (mass open cloud) Datacenter.
+*   All our deployment steps will be documented.
+
+
+## TOOLS
+
+There are two tools to accomplish our desire infrastructure:
+
+*   Using the cloud.redhat.com tool to deploy the cluster
+*   Using the Openshift install binary (KNI) to deploy the cluster (IPI).
+
+### Pros and Cons for cloud.redhat.com (Assisted Installer):
+**Pros:**
+
+- Assisted Installer is a growing RedHat tool to install a new cluster, getting family with it will have future benefits.
+- This tool will be part of the ACM and will be used in deploying new clusters via the ACM.
+- It's a good opportunity to provided feedback to the developers.
+- Can be easily automated, use API calls.
+
+**Cons:**
+- A dev preview tool, not completed product
+- This method includes booting from an image a BM, a more complicated way.
+
+### Pros and Cons for Openshift install (KNI):
+**Pros:**
+
+- The installation is closer to a disconnected environment, and it's a good practice.
+- This method uses an external tool that is available for all.
+- Lightweight, comfortable and available to the general public.
+
+We chose to deploy the cluster using the OpenShift install (KNI).
+
+## LIMITS
+
+We can’t use public cloud providers as a part of our global environment.
+
+
+## GOALS
+
+Our goal in this project is to create a centralized production openshift cluster to manage other openshift clusters and to have an easy deployment for new clusters for our users to use.
+
+We will be practicing the Operate First with several Red Hat products.
+
+
+## PROPOSED SOLUTION
+
+
+### High Level Architecture
+
+![image info](../images/Infra-cluster-overview.png)
+
+### Solution Overview
+
+
+
+*   **Openshift infra cluster -** a 3 Bare-metal nodes cluster is our main component.
+*   **Managed Cluster -** An OpenSHift cluster that is managed by the hub cluster. The hub will control the updates and resources on this cluster alongside application deployment.
+
+
+### Cluster Infra Design
+
+![image info](../images/Infra-cluster-zoom-in.png)
+
+
+### Cluster Infra Overview
+
```

Comment:
This section seems less like a "cluster infra overview" and more like a glossary (a list of definitions of words used in the document).